app/javascript/app/components/nav/nav-styles.scss

```diff
@@ -1,18 +1,101 @@
 @import '~styles/settings.scss';
 
-.navbar {
-  height: $header-height;
+$header-side-margin: 55px;
+$header-top-margin: 30px;
+$logo-width: 215px;
+$flag-height: 40px;
+
+// Header wrapper
+.header {
+  margin: {
```

Comment:
And I think we could think a better approach for the max-width of the page.
For example:
@import '~styles/layout.scss'; 
.header {
  @include row();
}

should do that work for us, but we can discuss it and find the best approach.

Reply:
👍 let's use the row, you're right